src/unicode.cpp

```diff
@@ -60,22 +60,20 @@ ada_really_inline bool has_tabs_or_newline(
   }
   // fast path for long strings (expected to be common)
   size_t i = 0;
-  const uint8x16_t mask1 = vmovq_n_u8('\r');
-  const uint8x16_t mask2 = vmovq_n_u8('\n');
-  const uint8x16_t mask3 = vmovq_n_u8('\t');
+  static uint8_t rnt_array[16] = {1, 0,  0, 0, 0,  0, 0, 0,
```

Comment:
I would also suggest to mention original work, to give credit to an author
https://gist.github.com/aqrit/6e73ca6ff52f72a2b121d584745f89f3#file-despace-cpp-L174

Reply:
I credit aqrit. I'd rather not link to a gist that may or may not be permanent.